api-put-object-copy.go

```diff
@@ -22,11 +22,14 @@ import (
 	"io"
 	"io/ioutil"
 	"net/http"
+	"time"
 
 	"github.com/minio/minio-go/v6/pkg/encrypt"
 	"github.com/minio/minio-go/v6/pkg/s3utils"
 )
 
+var timeSentinel = time.Unix(0, 0).UTC()
```

Comment:
So this converts unit time 0 at (server) local time to UTC... which will not be the same for all servers.